README.md

```diff
@@ -14,6 +14,7 @@ Convenience library to help write Kill Bill plugins.
 | 4.1.y             | 0.22.z            |
 | 4.2.y             | 0.22.z            |
 | 5.0.y             | 0.24.z            |
+| 5.1.y             | 0.24.z            |
 
 We've upgraded numerous dependencies in 4.2.x (required for Java 11 support).
```

Comment:
I would add a note like (or something along these lines):

```
Guava has been removed as a dependency in 5.1.x, native JDK alternatives should be used instead
```

Otherwise, if a user upgrades without looking at the code changes, they might get confused why the compilation errors if they implicitly rely on Guava in their plugins 😅

Reply:
We have this in [NEWS commit](https://github.com/killbill/killbill-plugin-framework-java/pull/91/files#diff-7ee66c4f1536ac84dc5bbff1b8312e2eef24b974b3e48a5c5c2bcfdf2eb8f3ce).